.github/workflows/ci.yaml

```diff
@@ -118,20 +115,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+
+        # 3.7 is to ensure we're safe for mlrun clients <= 1.3.0
+        # 3.9 is the current >= 1.3.0 python version
+        python-version: [3.7, 3.9]
```

Comment:
I guess 3.7 is relevant only for older branches such as 1.2.x which has their own `ci.yaml` and have the 3.7 option already configured. So if you already change the python version leave only the py39